src/quicklookUI.cs

```diff
@@ -168,5 +168,9 @@ interface QLPreviewingController {
 #endif
 		[Export ("preparePreviewOfSearchableItemWithIdentifier:queryString:completionHandler:")]
 		void PreparePreviewOfSearchableItem (string identifier, string queryString, Action<NSError> ItemLoadingHandler);
+
+		[Mac (10,15, onlyOn64: true)]
+		[Export ("preparePreviewOfFileAtURL:completionHandler:")]
+		void PreparePreviewOfFile (NSUrl url, Action<NSError> ItemLoadingHandler);
```

Comment:
I would remove the `OfFile` suffix but Swift kept it... not a huge thing :-)

But the second parameter should be `completionHandler`, not `ItemLoadingHandler` (it looks like a _copy/paste_ error in previous API declaration) as it should match the ObjC parameter name and **not* start with a capital letter.

Reply:
`completionHandler` makes a lot more sense - thank you! New API is updated.

Leaving previous API as-is to avoid diff noise unless otherwise advised :)